resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -686,22 +722,21 @@ addOreDictionaryTooltips=true
   </recipeGroup>
 
 
-  <recipeGroup name="Applied Energistics">
-
+  <recipeGroup name="Applied Energistics 2">
 
     <recipe name="ChargedCertusQuartzOre" energyCost="2400">
       <input>
-        <itemStack  modID="appliedenergistics2" itemName="tile.OreQuartzCharged"  />
+        <itemStack modID="appliedenergistics2" itemName="tile.OreQuartzCharged" />
       </input>
       <output>
-        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1"/>
+        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1" />
```

Comment:
Ok I've done some testing with mining:
1) Pickaxe without luck -> 1-2 crystals (no dusts)
2) Pickaxe with luck -> 1-6 crystals (no dusts)
So what is good value will be there? 3 crystals with one dust (10% chance)?